### PR TITLE
Add Flappy Dash Game

### PR DESCRIPTION
**Pull request by RutvikPanchal246**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,6 +189,8 @@ By Tommy Buonomo ([Frenchie Games](https://frenchiegames.app/#/))
 
 - Save The Potato [source-code](https://github.com/imaNNeo/save_the_potato) - 🥇 Winner of [Flame Game Jam 3.0](https://itch.io/jam/flame-jam-3/results) - Rotate the shields and save the potato from incoming orbs! By [imaNNeo](https://imanneo.dev). [for Android or iOS](http://savethepotato.com)
 
+- Flappy Dash - [Android](https://play.google.com/store/apps/details?id=games.pocket.flappydash.flappy_dash) - Gives classic flappy bird game play experience. By [Rutvik Panchal](https://twitter.com/rutvikpanchal_x)
```
Comment (ufrshubham) [inline README.md:192]: You can remove the entry from here and add the Play Store link in the open source section, similar to some of the other entries there.

Comment (RutvikPanchal246) [reply README.md]: Updated, Please proceed.

File: README.md [hunk 2]
```diff
@@ -188,6 +189,7 @@ By Tommy Buonomo ([Frenchie Games](https://frenchiegames.app/#/))
 
 - Save The Potato [source-code](https://github.com/imaNNeo/save_the_potato) - 🥇 Winner of [Flame Game Jam 3.0](https://itch.io/jam/flame-jam-3/results) - Rotate the shields and save the potato from incoming orbs! By [imaNNeo](https://imanneo.dev). [for Android or iOS](http://savethepotato.com)
 
+
```
Comment (spydon) [inline README.md:192]:
```suggestion
```

**Review by spydon — COMMENTED** [review]
The old flappy birds clone should also be removed

File: README.md (hunk 2 above)
Comment (RutvikPanchal246) [reply README.md:191]: @spydon Old flappy flame (flappy bird) game removed

Comment (ufrshubham) [reply README.md:191]: I think the suggestion is for removing the extra empty line

Comment (RutvikPanchal246) [reply README.md]: @ufrshubham Made changes as suggested

**Review by ufrshubham — APPROVED** [review]

**Review by spydon — APPROVED** [review]

**Comment by spydon** [comment]
One suggestion for the code, you don't need to use GetX for the little state you have, you can have it directly in the game class. :) 
We (Blue Fire) discourage the use of GetX.

**Comment by RutvikPanchal246** [comment]
@spydon Note down your feedback, thank you.

**Pull request merged**